src/com/serial/Test.java

```diff
@@ -0,0 +1,33 @@
+package com.serial;
+
+import java.io.*;
+import java.nio.file.Path;
+
+public class Test {
+
+    public static void main(String[] args) {
+        Path path = Path.of("res", "serialObj.txt");
+
+        serializeWriteObj(path);
+        deSerializeReadObj(path);
+    }
+
+    private static void serializeWriteObj(Path path) {
```

Comment:
It's not a popular way of creating methods, In work we use static methods only for some Utility methods (which isn't considered the best practice, and for some people, it's a sign of a bad application design) or for libraries like Java.math.calc() (it's not a precise path). Otherwise, you should create an object from the class in your main method, using private static is not a mistake, some old-school developers use it but we're learning how it should be not how it can be. So create an instance of the class and call it's methods in the main class, please.

Reply:
ok, I fixed problem